README.md

```diff
@@ -24,7 +24,13 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 
 ## Config Validation
 
-To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
+To validate a config for Ignition there are binaries for a cli tool called `ignition-validate` available [on the releases page][releases]. Three is also an ignition-validate container: `quay.io/coreos/ignition-validate`.
```

Comment:
Sorry, totally missed this the first time: s/Three/There/  :)

Reply:
Fixed